spec/core_functions/color/rgb/one_arg/alpha/expected_output.css

```diff
@@ -0,0 +1,18 @@
+basic {
+  transparent: rgba(0, 255, 127, 0);
+  opaque: #beaded;
+  partial: rgba(18, 52, 86, 0.5);
+  percent: rgba(18, 52, 86, 0.5);
+  named: rgba(0, 255, 127, 0.3);
+  parenthesized: rgba(0, 255, 127, 0.3);
+}
+
+clamped {
+  red: rgba(255, 0, 0, 0.5);
+  green: rgba(0, 0, 0, 0.5);
+  blue: rgba(0, 0, 255, 0.5);
+  alpha-above: black;
+  alpha-below: rgba(0, 0, 0, 0);
+  alpha-above-percent: black;
+  alpha-below-percent: rgba(0, 0, 0, 0);
```

Comment:
The "aboves" are clamped to 100% opacity, which means they're equivalent to `rgba(0, 0, 0)`, which is black. Sass renders it as `black` rather than e.g. `#000` because colors are generally more readable than hex codes.